.gitignore

```diff
@@ -19,3 +19,6 @@ node_modules/
 charts/.artifacts
 .cr-release-packages/
 .cr-index/
+
+# Idea
+.idea
```

Comment:
I'd prefer if we didn't have to include every possible IDE in the gitignore.
Please consider listing .idea in your [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) so we don't have to do this in every repo.
(I'm fine for it now, but I might later remove this line again as a side change in another PR :)

Reply:
It's good that you brought this up as I search a bit for a solution and couldn't find any. I also forgot to ask how you're doing it. I will remove it.